cmd/vulcan-http-headers/Dockerfile

```diff
@@ -1,12 +1,12 @@
 # Copyright 2019 Adevinta
 
-FROM python:3.6.4-slim-stretch
+FROM python:3.7.13-slim-bullseye
```

Comment:
Please upgrade to `python:3.7.13-slim-bullseye`

Reply:
fixed